src/Engine/StreamStorm/ai/Langchain.py

```
@@ -0,0 +1,27 @@
+from .LangchainModelFactory import ModelFactory
+from .Base import AIBase
+from .ResponseModels import AIResponse
+
+class LangchainAI(AIBase):
+    def __init__(self, provider_name: str, model_name: str, api_key: str, base_url: str = None):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)